.eslintrc

```diff
@@ -61,6 +61,7 @@
       "objectsInObjects": false,
       "arraysInObjects": false
     }],
+    "object-shorthand": [2, "properties"],
```

Comment:
@erwinmombay What do you think about the error level? I found that travis would still fail if it sees lint warnings. So we might make it `error` level? Do you think I should make the lint rule a separate PR?